.travis.yml

```diff
@@ -12,5 +12,5 @@ if: (type == push) OR (type == pull_request AND fork == true)
 
 script:
   - flake8
-  # - coverage run --source drs-filer -m pytest
-  # - coveralls
\ No newline at end of file
+  # - coverage run --source drs_filer -m pytest
```

Comment:
Sure.
Now, if I add this, I suspect it will cause the build to fail because there are no tests after all. So, will add it later when I add some basic tests.